custom-plugin-decorators/sorting/sort-tags.js

```diff
@@ -0,0 +1,16 @@
+module.exports = SortTagsAlphabetically;
+
+function SortTagsAlphabetically() {
+  console.log("re-ordering tags: alphabetical");
+  return {
+    TagList: {
+      leave(target) {
+        target.sort((a,b) => {
+          if (a.name < b.name) {
+            return -1;
+          }
+        });
```

Comment:
```suggestion
        target.sort((a, b) => (a.name < b.name ? -1 : a.name > b.name ? 1 : 0));
```

Reply:
In favour of readability (this code is meant to be read and understood by all users) I don't think this is an improvement.